### DISCUSSION TITLE
Importing authorization rules - question

### DISCUSSION THREAD

**Discussion by sylwester-majcher** [lithnet/access-manager#84] [Q&A]
I saw few options to import authorization rules. there is the option on this page to save actual rules. is there possibility to simply use this saved file to import rules? I have tried but no success. I think about sceneroius to migrate LAM to another sever for example

**Comment by ryannewington** [comment]
Are you talking about the "Save permission report..." option?

That doesn't quite have enough of the details needed, but we could look at adding functionality to import rules from one AMS instance to another, if thats what you are after?

As a workaround, you can open the appsettings.json file and copy the Authorization/ComputerTargets sections from one file to another, provided of course both instances are from the same domain.

**Reply by sylwester-majcher** [reply]
Right, I am talking about "Save permission report...".
I am tasting workeround and it looks good so in my opinion maybe only you can add some short information inside the Wiki about this solution and that's all :) thanks. for me subject to close :)
